.gitignore

```diff
@@ -138,3 +138,4 @@ extensions
 
 # templates from widdershins
 templates/openapi3
+/.vs
```

Comment:
what's this?

Reply:
sorry, it is my local change, removed it